packages/salesforce/package.json

```diff
@@ -28,25 +28,25 @@
   ],
   "dependencies": {
     "@openfn/language-common": "1.7.3",
+    "JSONPath": "^0.10.0",
     "axios": "^0.21.1",
     "jsforce": "^1.11.0",
-    "JSONPath": "^0.10.0",
     "lodash-fp": "^0.10.2",
     "mustache": "^2.2.0",
+    "sinon": "^14.0.1",
```

Comment:
This needs to be a dev dependency! That's my fault if you used the command I gave you...

Reply:
@josephjclark fixed https://github.com/OpenFn/adaptors/pull/105/commits/c24beb8b18837a953e1c10cdba5f67fbf8f79ca5